compiler/rustc_lint/src/builtin.rs

```diff
@@ -303,11 +311,11 @@ declare_lint! {
     ///
     /// ### Explanation
     ///
-    /// This lint is intended to restrict the usage of `unsafe`, which can be
-    /// difficult to use correctly.
+    /// This lint is intended to restrict the usage of `unsafe`,
+    /// and `no_mangle`, which can be difficult to use correctly.
```

Comment:
this only mentions `no_mangle`, while the top says "other potentially unsound constructs like `no_mangle`, `export_name`, and `link_section`", I think this should be expanded to much the top?

Reply:
Updated. But I'm unsure if I comprehended your meaning.